xbmc/pvr/filesystem/PVRGUIDirectory.cpp

```diff
@@ -100,6 +103,21 @@ bool GetRootDirectory(bool bRadio, CFileItemList& results)
     results.Add(item);
   }
 
+  // Media
+  const auto& media = CServiceBroker::GetPVRManager().Media();
+  if (clients->AnyClientSupportingMedia() &&
+      ((bRadio && media->GetNumRadioMedia() > 0) || (!bRadio && media->GetNumTVMedia() > 0)))
+  {
+    item.reset(new CFileItem(bRadio ? CPVRMediaPath::PATH_ACTIVE_RADIO_MEDIA
+                                    : CPVRMediaPath::PATH_ACTIVE_TV_MEDIA,
+                             true));
+    item->SetLabel(g_localizeStrings.Get(14211)); // Media
+    item->SetProperty("node.target", CWindowTranslator::TranslateWindow(bRadio ? WINDOW_RADIO_MEDIA
+                                                                               : WINDOW_TV_MEDIA));
+    item->SetArt("icon", "DefaultPVRRecordings.png"); // TODO: Update Image
```

Comment:
@ronie, here is where I use the Recordings icon as I didn't have one for Media. Media is similar to Recordings except you can't create them from timers and they can't be deleted by default.